staging/src/k8s.io/api/admission/v1beta1/types.go

```diff
@@ -94,6 +94,10 @@ type AdmissionResponse struct {
 	// The type of Patch. Currently we only allow "JSONPatch".
 	// +optional
 	PatchType *PatchType `json:"patchType,omitempty" protobuf:"bytes,5,opt,name=patchType"`
+
+	// Annotations set by remote admission controller.
```

Comment:
there should be a specified format. What are the keys?

Reply:
This Annotation is a little bit different from annotation in admission attribute.
It doesn't contain the fully qualified plugin name. 

I use [this plugin name](https://github.com/kubernetes/kubernetes/blob/f3942e7dedc0781aaee0a8dec1e134748b705ae2/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L130) to call attributesRecord.SetAnnotations().

Reply:
Done.